src/program/lwaftr/run_nohw/run_nohw.lua

```diff
@@ -62,9 +62,12 @@ function run(parameters)
    local conf_file, b4_if, inet_if, opts = parse_args(parameters)
    local conf = require('apps.lwaftr.conf').load_lwaftr_config(conf_file)
    local c = config.new()
+   local lwaftr_class =  LwAftr
+   lwaftr_class.config_arg = "conf"
+   local device = next(assert(conf.softwire_config.instance))
 
    -- AFTR
-   config.app(c, "aftr", LwAftr, conf)
+   config.app(c, "aftr", lwaftr_class, {conf=conf, device=device})
```

Comment:
Hummmmmmmmmmmm.  This seems like a perilous change to make.  Right now the configuration of the lwaftr app is described by the snabb-softwire-v2 schema, and that's how it gets shipped over the wire.  Making it ad-hoc is going in the wrong direction, I think....